README.md

```diff
@@ -1,14 +1,14 @@
 # pg-schema-diff
+Diffs Postgres database schemas and generates the SQL required to get your database schema from point A to B while 
```

Comment:
Diff is such a weird verb that I feel like it's hard to use in the first sentence of a doc. Maybe this?
```suggestion
Computes a diff of Postgres database schemas and generates the SQL required to get your database schema from point A to B while 
```